frontend/src/components/Login.js

```diff
@@ -21,6 +21,7 @@ class Login extends Component {
     this.handleUserChange = this.handleUserChange.bind(this);
     this.goToRegistration = this.goToRegistration.bind(this);
     this.login = this.login.bind(this);
+    this.renderErrors = this.renderErrors.bind(this); 
```

Comment:
There is a stray space at the end of this line. I know there are a lot of code quality issues we have. :) But let's start to pay attention to these when we write new code